tests/php/LeftAndMainTest.php

```diff
@@ -108,6 +116,7 @@ public function testLeftAndMainSubclasses()
             $this->assertRegExp('/<head[^>]*>/i', $response->getBody(), "$link should contain <head> tag");
             $this->assertRegExp('/<body[^>]*>/i', $response->getBody(), "$link should contain <body> tag");
         }
+        $this->assertSame(count($itemsToTest), $tested, 'All of the menu items we want to test were tested');
```

Comment:
Hmmm - this is kind of gross, but maybe you should do:

`$this->assertCount($tested, $itemsToTest)`? Also, *the message is for printing when there's been an error* so  something like "Not all menu items were successfully tested"

Reply:
Not sure - the expected value should come first

Reply:
Yeah, true....